File: src/main/java/easy/ContainsDuplicate.java
package easy;

import java.util.HashSet;
import java.util.Set;

public class ContainsDuplicate {

    public boolean containsDuplicate(int[] nums) {
        Set<Integer> integerSet = new HashSet<>();

        for (int num : nums) {
            if (!integerSet.add(num)) {
                return true;
            }
        }

        return false;
    }

    public static void main(String[] args) {
        ContainsDuplicate cd = new ContainsDuplicate();
        int[] nums = {1, 2, 1, 4, 6};
        boolean result = cd.containsDuplicate(nums);
        System.out.println(result);
    }
}

File: src/main/java/easy/kotlin/ContainsDuplicate.kt
package easy.kotlin

class ContainsDuplicate {

    fun containsDuplicate(nums: IntArray): Boolean {
        val integerSet = mutableSetOf<Int>()

        for (num in nums) {
            if (!integerSet.add(num)) {
                return true
            }
        }

        return false
    }
}